.kokoro/common.cfg

```diff
@@ -11,3 +11,12 @@ env_vars: {
   key: "TRAMPOLINE_BUILD_FILE"
   value: "github/java-retail/.kokoro/build.sh"
 }
+env_vars: {
+  key: "BUCKET_NAME"
+  value: "crs-interactive-tutorials"
+}
+
+env_vars: {
+  key: "EVENTS_BUCKET_NAME"
+  value: "crs-interactive-tutorials"
+}
```

Comment:
These kokoro files are templated, so you probably want to rely on this file to populate secrets: https://github.com/googleapis/java-retail/blob/main/.kokoro/populate-secrets.sh

Reply:
done